docs/dataset-formats/conversation.qmd

```diff
@@ -191,9 +191,7 @@ Instead of passing `tools` via the system prompt, an alternative method would be
                     "type": "function",
                     "function": {
                         "name": "...",
-                        "arguments": {
-                            "...": "...",
-                        }
+                        "arguments": "{\"...\": \"...\"}" // should be json string
```

Comment:
This seems to be an edge case, so we can revert this and put a NOTE / TIP at the bottom that for those with exact name keys but diff types to use json string instead.

Otherwise, users with working datasets may also try to convert it when unneeded.